virtumall/settings.py

```diff
@@ -12,14 +12,21 @@
 
 import os
 from pathlib import Path
+from typing import Any
+from typing import List
+from typing import Optional
+from typing import Union
 
 from django.core.exceptions import ImproperlyConfigured
-from dotenv import find_dotenv, load_dotenv
+from dotenv import find_dotenv
+from dotenv import load_dotenv
 
 load_dotenv(find_dotenv())
 
 
-def get_env_variable(env_key, default=None):
+def get_env_variable(
+    env_key: str, default: Optional[Any] = None
+) -> Union[str, bool, Any]:
```

Comment:
```suggestion
) -> str | bool | Any:
```

3.9 이상 파이썬에서는 이런 형태가 나을 거에요 ㅎㅎ